data/heroes/purple/5star.yml

```diff
@@ -344,6 +344,23 @@
     - Mana Gain Freeze
   family: 2017
   image: https://i.imgur.com/5oaRVxv.jpg
+  costume:
+    class: Cleric
+    power: 765
+    attack: 745
+    defense: 692
+    health: 1328
+    skill: Crush of Malice
+    effects:
+      - Deals 300% damage to all enemies.
+      - The caster steals generated mana of all enemies over 4 turns starting low and increasing every turn until on the last turn all generated mana is stolen. This effect can’t be cleansed, but it gets removed if the caster dies.
+      - Deals extra damage to Holy.
+      - Elemental Link gives all Dark allies +15% attack and +15% defense for 6 turns. This effect can’t be displelled.
+    types:
+      - Hit 5
+      - Mana Steal
+    image: https://i.imgur.com/l8I8LCr.jpg
```

Comment:
Looks like the 3 star template was used instead of the 5.  Can you update it to be the 5 star template?